opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -553,6 +557,16 @@ def set_attribute(self, key: str, value: types.AttributeValue) -> None:
             with self._lock:
                 self.attributes[key] = value
 
+    def get_attribute(self, key: str) -> types.AttributeValue:
+        if not key:
+            logger.warning("invalid key (empty or null)")
+
+        if key not in self.attributes:
+            logger.warning("no attributes for key:%s", str(key))
```

Comment:
If you don't return `None` here, I believe there will be a `KeyError` raised on L568.

Reply:
Did not see that thank you.